README.md

```diff
@@ -0,0 +1,116 @@
+# IntegratorXX
+
+## Synopsis
+
+IntegratorXX is a modern C++ library for the generation of atomic and molecular
+grids for quantum chemistry calculations. It provides a uniform interface for
+the generation of primitive, radial and solid angle quadratures, as well as
+there combination into spherical grids.
```

Comment:
```suggestion

IntegratorXX is a modern C++ library for the generation of atomic and molecular
grids for quantum chemistry calculations. The most important use of these
quadratures is the calculation of the exchange-correlation energy in density
functional theory

$$ E_{\text{xc}}=\int f_{\text{xc}}(n,\nabla n,\nabla^{2}n,\tau){\rm d}^{3}r $$

using the multi-center approach pioneered by Becke in [J. Chem. Phys. 88,
2547–2553 (1988)](https://doi.org/10.1063/1.454033); however, the
quadratures are also useful in many other contexts.

IntegratorXX provides a uniform interface for the generation of 
primitive, radial and solid angle quadratures, as well as
their combination into spherical grids.
```

Reply:
I don't think it's a good idea to pin ourselves to XC-related topics - as you've said yourself, grids like these are required to evaluate _**everything**_ in other basis expansions. You want to list some examples, that's fine, but going into things like the Becke scheme for the purposes of a README is a bit pedantic IMHO.

Reply:
Well yes, but given that XC is going to be 99% of our usage, I think it would be silly not to mention it here :)